api/src/test/java/org/openmrs/AllergiesTest.java

```diff
@@ -17,17 +17,30 @@
 import org.junit.Test;
 import org.openmrs.Concept;
 import org.openmrs.api.APIException;
+import org.openmrs.util.OpenmrsConstants;
+import org.openmrs.api.context.Context;
+import org.openmrs.test.BaseContextSensitiveTest;
 
 /**
  * Tests methods in {@link org.openmrs.Allergies}.
  */
-public class AllergiesTest {
+public class AllergiesTest extends BaseContextSensitiveTest {
+	
+	private static final String ALLERGY_OTHER_NONCODED_TEST_DATASET = "org/openmrs/api/include/otherNonCodedConcept.xml";
 	
 	Allergies allergies;
+	private String otherNonCodedConceptUuid;
```

Comment:
In this class, the same comments as above apply for the local variable otherNonCodedConceptUuid